_posts/ja/posts/2018-06-15-release-0.16.1.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Bitcoin Core 0.16.1 リリース
+name: blog-release-0.16.1
+id: ja-blog-release-0.16.1
+lang: ja
+permalink: /ja/2018/06/15/release-0.16.1/
+type: posts
+layout: post
+share: true
+version: 1
+
+excerpt: >
+  最新のBug Fixとマイナーアップデートを含むBitcoin Core 0.16.1がリリースされました
+---
+Bitcoin Core バージョン0.16.1を[ダウンロード][ダウンロードページ]できるようになりました。
+すべてのユーザーが、いくつかのバグを修正し以下の新しいマイナー機能のバックポートを提供する
+この[メンテナンスリリース][]にアップグレードすることをお勧めします。
+
+- DoS攻撃の攻撃ベクトルを軽減します。この攻撃は特定のサービスを犠牲にし、
+おそらく（ネットワークに接続して数時間以上経過しているノードよりも）初めて起動したノードに対して最も効果的だったでしょう。
+
+- マイナーが非常に迅速に連続して2つのブロックを生成した場合に、報酬を失う可能性があるバグを修正しました。
+
+- レガシーな（非segwitの）署名スクリプトで、ほとんど見られない`OP_CODESEPARATOR`
+opcodeが使われている場合、そのトランザクションのリレーを停止します。このopcodeが現れると
+ノードがレガシーな署名スクリプトを検証するためにどれだけの計算作業が必要か推定することが困難になります。
+このため、攻撃者が検証に長時間かかるブロックを作成できないように、このソリューションの展開をブロックします。
```

Comment:
I fixed it so that it becomes clearer.
`このため、OP_CODESEPARATORは攻撃者に検証に長時間かかるブロックを作成させないようにするソリューションの展開をブロックします。このリレーポリシーの変更は問題自体を解決するものではありませんが、ユーザーがこれらの採用に同意すると将来的に提案されたこれらのソリューションを安全かつ簡単に展開できます。`